config/config.go

```diff
@@ -170,9 +180,9 @@ type ProxyConfig struct {
 	ComponentConfig
 	Endpoint string   `json:"endpoint" required:"true"`
 	Timeout  Duration `json:"timeout"`
-	Protocol string   `json:"protocol"`
-	Service  string   `json:"service"`
-	Method   string   `json:"method"`
+	Protocol string   `json:"protocol,omitempty"`
```

Comment:
Why is this `omitempty`? I think it would be clearer to force the user to set this value instead of defaulting to http. We can update any depending projects (like Turing) accordingly, when we consume the latest Fiber changes.